### ISSUE TITLE
Feature Request: Add timeout to the aws_glue_job resource

### ISSUE THREAD

**Issue by TLipede** [terraform-providers/terraform-provider-aws#4332]
### Terraform Version
Terraform v0.11.7

### Affected Resource(s)
- aws_glue_job

### Terraform Configuration Files
```hcl
resource "aws_glue_job" "glue-jobs" {
  count = "${length(var.glue_jobs)}"

  name               = "${var.glue_jobs[count.index]}"
  allocated_capacity = 10
  execution_property = {
    max_concurrent_runs = 1
    timeout             = 120
  }
  role_arn           = "${aws_iam_role.glue-jobs.arn}"
}
```

### Expected Behavior
Timeout values were very recently added as a feature for AWS Glue jobs, and I would like to add this as an execution property.

### References
https://aws.amazon.com/about-aws/whats-new/2018/04/aws-glue-now-supports-timeout-values-for-etl-jobs/
https://docs.aws.amazon.com/glue/latest/dg/add-job.html

**Comment by bflad** [comment]
This will be released later today with v1.18.0 of the AWS provider.

**Comment by bflad** [comment]
This has been released in [version 1.18.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1180-may-10-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
